Sources/AnimalKingdomAPI/GeneratedCCN/Operations/AllAnimalsQueryCCNQuery.swift

```diff
@@ -0,0 +1,63 @@
+// @generated
```

Comment:
@AnthonyMDev I changed the name of the query, and it looks like the old query was kept around. idk if you're already aware of this bug.

Reply:
Do you mean a file with the old query name was still present? If so then it's not a 'bug' as we haven't coded for it to clean up, not sure how we'd do that yet.

Reply:
Yeah, you need to handle that manually currently. We may do something about it in the future, but it's not a priority at the moment.